tests/server/utils.py

```diff
@@ -6,20 +6,24 @@
 
 from fastapi.testclient import TestClient
 
+from optimade import __api_version__
+
+VERSION_PREFIX = f"/v{__api_version__.split('.')[0]}"
+
 
 def get_regular_client() -> TestClient:
     """Return TestClient for regular OPTIMADE server"""
     from optimade.server.main import app
     from optimade.server.routers import info, links, references, structures
 
     # We need to remove the version prefixes in order to have the tests run correctly.
-    app.include_router(info.router)
-    app.include_router(links.router)
-    app.include_router(references.router)
-    app.include_router(structures.router)
+    app.include_router(info.router, prefix=VERSION_PREFIX)
+    app.include_router(links.router, prefix=VERSION_PREFIX)
+    app.include_router(references.router, prefix=VERSION_PREFIX)
+    app.include_router(structures.router, prefix=VERSION_PREFIX)
```

Comment:
To match your new setup in `main.py` :)
```suggestion
    for endpoint in (info, links, references, structures):
        app.include_router(endpoint.router, prefix=VERSION_PREFIX)
```